.snyk

```diff
@@ -0,0 +1,6 @@
+ignore:
+  MPL-2.0
+    - reason: 'MPL-2.0 is compatible with Rook licenses'
```

Comment:
This doesn't match the ignore config I see in snyk docs https://docs.snyk.io/snyk-cli/commands/ignore

Is this a special case that doesn't require the `PATH_TO_RESOURCE` component?

Reply:
@BlaineEXE  wrote:
> This doesn't match the ignore config I see in snyk docs https://docs.snyk.io/snyk-cli/commands/ignore
> 
> Is this a special case that doesn't require the `PATH_TO_RESOURCE` component?

not sure.
The link you provided is  to documentation of the `snyk ignore` command which can apparently be used to add ignores to the `.snyk` file. My main problem with this is that the doc mentions a "snyk vulnerability id" that one has to specify, but I don't  know how to find the appropriate id. this might be a reason for not being able to progress here ...

that said, th ci still fails with the note that `snyk auth` needs to be run which is strange since the `SNYK_TOKEN` is used in thr eorkflow file ....

Reply:
[This doc](https://docs.snyk.io/manage-risk/policies/the-.snyk-file#ignoring-the-license-with-the-cli) and the link given in the failure in CI (https://snyk.io/vuln/snyk:lic:golang:github.com:hashicorp:errwrap:MPL-2.0) suggests that `MPL-2.0` could be the right value, or it could be on a per-import basis like `snyk:lic:golang:github.com:hashicorp:errwrap:MPL-2.0`, but I'm unsure.

Reply:
The doc has this example config:
```
ignore:
  '<ISSUE_ID>':
    - '<PATH_TO_RESOURCE>':
        reason: <REASON>
        expires: <EXPIRY>
```

Look closely at how this compares to what is in your file. `PATH_TO_RESOURCE` is not captured. This change skips that header and jumps straight to `reason`